src/NuGet.Core/NuGet.Configuration/Settings/Items/CertificateItem.cs

```diff
@@ -0,0 +1,115 @@
+// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Xml.Linq;
+using NuGet.Common;
+using NuGet.Shared;
+
+namespace NuGet.Configuration
+{
+    public sealed class CertificateItem : SettingItem, IEquatable<CertificateItem>
+    {
+        public string Fingerprint => Attributes[ConfigurationConstants.Fingerprint];
+
+        public HashAlgorithmName HashAlgorithm => CryptoHashUtility.GetHashAlgorithmName(Attributes[ConfigurationConstants.HashAlgorithm]);
+
+        public bool AllowUntrustedRoot
+        {
+            get
+            {
+                if (bool.TryParse(Attributes[ConfigurationConstants.AllowUntrustedRoot], out var parsedAttribute))
```

Comment:
isn't `AllowUntrustedRoot` always be there for Certificate item?

Reply:
yes, but it might be the case that a user hand edited the file and added something different than `true` or `false`, therefore I want to make sure I parse it correctly.